src/gui/src/CapriceOptions.cpp

```diff
@@ -247,6 +247,7 @@ CapriceOptions::CapriceOptions(const CRect& WindowRect, CWindow* pParent, CFontE
     m_pDropDownCPCLanguage->SelectItem(CPC.keyboard);
     m_pDropDownCPCLanguage->SetIsFocusable(true);
     // option 'kbd_layout' which is the platform keyboard layout (i.e. the PC keyboard layout)
+    /*
```

Comment:
Adding a file is not an issue (it will not be available in options, that's all).
Removing one could be though (well, not even a big one, it will default to UK I think). But if someone mess-up with resources while running the program, he is looking for problems isn't he ?